sdk/rust/oak_config_serializer/src/main.rs

```diff
@@ -87,35 +87,55 @@ impl Default for InitialNodeConfig {
     }
 }
 
+/// Directory used to save downloaded Wasm modules.
+/// Created in the `std::env::current_dir()`.
+const CACHE_DIRECTORY: &str = ".oak";
+
 /// Computes SHA256 sum from `data` and returns it as a HEX encoded string.
 fn get_sha256(data: &[u8]) -> String {
     let mut hasher = Sha256::new();
     hasher.update(data);
     hex::encode(hasher.finalize().as_slice().to_vec())
 }
 
-/// Load Wasm module from URL or a file, if URL is not specified.
+/// Download file from `url`.
+async fn download_module_from_url(url: &str) -> anyhow::Result<Vec<u8>> {
```

Comment:
I think also the fn name and the error messages should talk about files, not modules.